sycl/test-e2e/Matrix/joint_matrix_query_default.cpp

```diff
@@ -12,21 +12,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-#include <iostream>
-#include <sycl/sycl.hpp>
-
-using namespace sycl;
-using namespace sycl::ext::oneapi::experimental::matrix;
-
-template <typename T, size_t NUM_ROWS, size_t NUM_COLS> struct big_matrix {
-public:
-  T *mat;
-
-public:
-  T *get_data() { return mat; }
-  void set_data(T *data) { mat = data; }
-  big_matrix(T *data) : mat(data) {}
-};
+#include "common.hpp"
```

Comment:
Also CPU specific tests, needs to be verified on CPU

Reply:
Passed see below.

Reply:
I don't see this test was run at all.

Reply:
See response above.